changelogs/fragments/1239-lambda_info-return_list.yml

```diff
@@ -0,0 +1,8 @@
+minor_changes:
+- lambda_info - add return key ``lambda_info`` which returns a list of dictionaries instead of the previously returned
```

Comment:
Based on https://github.com/ansible-collections/amazon.aws/pull/865

This should probably be "functions" instead of "lambda_info"